decentraland/unity-renderer
# feat: [SUB-BRANCH] Create a confirmation modal for leave a channel

## What does this PR change?
Create a confirmation modal to be showed each time the user want to leave a channel:

![image](https://user-images.githubusercontent.com/64659061/183905098-f1e67d6d-4745-41f7-bc30-ff601e534517.png)

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=feat/confirmation-modal-for-leave-a-channel
2. Try to leave a channel (from the conversation list, from a specific channel window or from the search channels window).
3. Notice that a new confirmation modal is open and let you confirm or cancel the leave.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
